README.md

```diff
@@ -1,2 +1,149 @@
+   uploaded, they need to be encrypted with the correct public key. This key can
+   be downloaded from this repository with this command:
+   ```bash
+   wget https://raw.githubusercontent.com/NBISweden/EGA-SE-user-docs/main/crypt4gh_key.pub
+   ```
+
+ - (Optional) Create a personal key pair​
+
+   The creation of a key pair is very simple using crypt4gh. Run the following
+   command, replacing [my-key] with the name of the key (can be anything) and
+   specifying a passphrase when requested:
+​
+   ```bash
+   crypt4gh-keygen --pk [my-key].pub --sk [my-key].sec
+   ```
+
+   To verify that the key pair was created, run the `ls` command and make sure
+   the keys you specified exist in the folder.
+
+ - Encrypt the files
+
+   Now that you have the public key, and the tools you need, you can encrypt the
+   submission files. An encryption key will be created automatically by the
+   tool, but if you prefer to use a specific key, you can provide one using the
+   `--sk` argument.
+
+   **Warning:** Please make sure to use different names (at least different
+   extensions) for `[my-file]` and `[my-encrypted-file]` to avoid destroying
+   your original file!
+
+   with singularity:
+   ```bash
+   ./crypt4gh encrypt --recipient_pk crypt4gh_key.pub < [my-file] > [my-encrypted-file].c4gh
+   ```
+
+   with native tools:
+   ```bash
+   ./crypt4gh encrypt --recipient_pk crypt4gh_key.pub < [my-file] > [my-encrypted-file].c4gh
+   ```
+
+## Submitting
+
+Once your files are encrypted you are almost ready to start submitting to the
+SDA. There is just one more thing that is needed; checksums for the files.
+
+### Get checksums of the files
+​
+Once the files are uploaded they need to be validated, which will require you to enter md5 and sha256 checksums for each file. To prepare for this later step, it's adviced to create checksums for all files as you upload them.
+
+Check our short guide on [Calculating checksums of files](checksums.md) for details.
+​
+### Get the configuration file
+​
+The s3cmd tool requires a configuration file with the relevant settings. You
+can get the configuration file by logging in with your Elixir ID
+[here](https://login.sda.nbis.se/).
+
+If you choose not to use the downloaded configuration file, we recommend
+setting the multipart chunk size significantly higher than the default 5 Mbyte.
+It can be set up to 2 Gbytes but values above 100 Mbyte probably do very little
+to improve throughput.
+
+### Upload the file(s)​
+
+S3 allows for **optional** creation of folders and that can be achieved by
+defining the path in the my-s3-path variable.
+
+```bash
+s3cmd -v -c [my-path-to-s3-config] put [my-encrypted-file].c4gh "s3://[my-username]/[my-s3-path]/[my-encrypted-file].c4gh"
+```
+​
+for example, if the file should be stored under the "experiment1" folder, the
+command would look like:
+​
+```bash
+s3cmd -v -c /path/s3-config put file1.c4gh "s3://user/experiment1/file1.c4gh"
+```
+​
```

Comment:
Maybe mention as well how you create a folder? I guess that is too much of detail to add that but it will be convenient for some researchers that are not familiar with the s3cmd